dags/oup/parser.py

```diff
@@ -53,3 +58,42 @@ def _get_journal_doctype(self, article: ET.Element):
                 journal_doctype=journal_doctype,
             )
         return journal_doctype
+
+    def _get_authors(self, article: ET.Element):
+        contributions = article.findall(
+            "front/article-meta/contrib-group/contrib[@contrib-type='author']"
+        )
+        authors = []
+        for contribution in contributions:
+            surname = get_text_value(contribution.find("name/surname"))
+            given_names = get_text_value(contribution.find("name/given-names"))
+            email = get_text_value(contribution.find("email"))
+            affiliations = contribution.findall("aff")
+
+            full_affiliation = [
+                {
+                    "institution": get_text_value(
+                        affiliation.find(
+                            "institution",
+                        )
+                    ),
+                    "country": get_text_value(
+                        affiliation.find(
+                            "country",
+                        )
+                    ).capitalize(),
+                }
+                for affiliation in affiliations
+            ]
+            if not surname and not given_names and not email and not full_affiliation:
```

Comment:
`if not all([surname, given_names, email, full_affiliation]):` won't work, because is an array of dicts, which will give an error ` TypeError: 'builtin_function_or_method' object is not subscriptable`

Reply:
well, it should work with dicts:
```
>>> a = {'test': 2}
>>> b = {'another': 3}
>>> c = {'yet': 5}
>>> all([a, b, c])
True
```
you need two brackets, it occurs when you add just a square bracket
```
>>> all[a, b, c, 7]
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'builtin_function_or_method' object is not subscriptable
```